tests/test_png.py

```diff
@@ -0,0 +1,109 @@
+from PIL import Image
+import os
+import urllib.request
+from fastapi import APIRouter
+from pims.formats import FORMATS
+import io
+from pims.importer.importer import FileImporter
+
+from pims.files.file import (
+    EXTRACTED_DIR, HISTOGRAM_STEM, ORIGINAL_STEM, PROCESSED_DIR, Path,
+    SPATIAL_STEM, UPLOAD_DIR_PREFIX, unique_name_generator
+)
+
+def getImage(path, image):
+	filepath = os.path.join(path, image)
+	# If image does not exist locally -> download image
+	if not os.path.exists(path):
+		os.mkdir("/data/pims/upload_test_png")
+	
+	if not os.path.exists(filepath):
+		try:
+			url = f"https://doc.cytomine.org/images/{image}" #OAC
+			urllib.request.urlretrieve(url, filepath)
+		except Exception as e:
+			print("Could not download image")
+			print(e)
+	
+	if not os.path.exists(path + "processed"):
```

Comment:
There is `os.path.join` to join folders :)

Reply:
Done